app/api/tax.py

```diff
@@ -66,10 +69,33 @@ def before_create_object(self, data, view_kwargs):
 
 
 class TaxDetail(ResourceDetail):
+    """
+     tax detail by id
+    """
+    def before_get_object(self, view_kwargs):
+        if view_kwargs.get('event_identifier'):
```

Comment:
event.id should not be view_kwargs['id'].Also handle this in same code pattern as done below in Line 80 for `event_id`